PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/layers/transaction_layer.cpp

```diff
@@ -0,0 +1,32 @@
+#include "transaction_layer.h"
+#include "Tlp.h"
```

Comment:
The file is called tlp.h and it's in utils. Are you sure this is building?

Reply:
`#include "tlp.h"` is not needed anymore as we agreed on making only the data, and header consumptions as a parameters instead of the whole TLP 👀 
https://grad-project-pcie.atlassian.net/browse/PCIE-87?focusedCommentId=10016